pom.xml

```diff
@@ -247,6 +247,14 @@
       <scope>test</scope>
     </dependency>
 
+    <dependency>
+      <groupId>org.jacoco</groupId>
+      <artifactId>org.jacoco.agent</artifactId>
+      <classifier>runtime</classifier>
+      <version>0.7.5.201505241946</version>
```

Comment:
also I suspect that number of duplicates can be reduced from 3 to 2 by declaring plugin in pluginManagement section